### DISCUSSION TITLE
HowTo Setup?

### DISCUSSION THREAD

**Discussion by GrtDlks** [iluminar/goodwork#1481] [Q&A]
Hey, im struggling about, how to set it up.
Every time I want to install the packages with composer I got this errors:

Package league/uri-parser is abandoned, you should avoid using it. No replacement was suggested.
Package tgalopin/html-sanitizer is abandoned, you should avoid using it. Use symfony/html-sanitizer instead.

**Comment by Hasnayeen** [comment]
This are warning and should not prevent from installation. can you share a screenshot of the error?

**Reply by GrtDlks** [reply]
I think a quick how to is better to post all screenshots of every error I got 😄

**Reply by Hasnayeen** [reply]
project is under heavy development so things will break heavily that's why not including any setup info in the readme. I'll add a proper readme when filament v3 reach stable release as this is based on filament.
